dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/MasterSchedulerBootstrap.java

```diff
@@ -232,6 +247,24 @@ private List<ProcessInstance> command2ProcessInstance(List<Command> commands) th
         return processInstances;
     }
 
+    private void sendRpcCommand(ProcessInstance processInstance) {
```

Comment:
Suggest move this method in `handleCommand`, otherwise, we need to execute sendRpcCommand in all other place when we  execute handleCommand.

We can just split handleCommand into two method: `handleCommandInDB` and `sendRpcCommand`, handleCommandInDb can have transaction.

Reply:
Good idea, i will change it.